x/pki/module.go

```diff
@@ -4,7 +4,7 @@ import (
 	"encoding/json"
 	"fmt"
 
-	// this line is used by starport scaffolding # 1
+	"context"
```

Comment:
could you please rename `capability` word (module name) with `pki`? (looks like `starport` has a minor issue and uses the name inside a template)

Reply:
Renamed, thanks.